package.js

```diff
@@ -1,48 +1,54 @@
+/* eslint-disable no-undef */
+
 Package.describe({
-  name: "mizzao:user-status",
-  summary: "User connection and idle state tracking for Meteor",
-  version: "0.6.8",
-  git: "https://github.com/mizzao/meteor-user-status.git"
+  name: 'mizzao:user-status',
+  summary: 'User connection and idle state tracking for Meteor',
+  version: '0.7.0',
+  git: 'https://github.com/mizzao/meteor-user-status.git'
 });
 
-Package.onUse( function(api) {
-  api.versionsFrom("1.2.0.1");
+Package.onUse((api) => {
+  api.versionsFrom('1.2.0.1');
 
+  api.use('ecmascript');
```

Comment:
With `ecmascript` package we should bump the minimum version of Meteor to at least 1.3.